README.md

```diff
@@ -108,14 +123,66 @@ travis encrypt --pro AZURE_WEBHOOK=$WEBHOOK
 ```
 
 #### Secure DB
+
 Take env vars from above as required. This uses IP firewall. Can't seem to do VNets (Tier S1 and above required it seems?)
+
 ```sh
 az webapp show --resource-group $RG --name $APPNAME --query outboundIpAddresses --output tsv # Call this and add the IPs to the azure portal
 ```
 
-
 #### Add
+
 Take env vars from above as required. This uses IP firewall. Can't seem to do VNets (Tier S1 and above required it seems?)
+
 ```sh
 az webapp show --resource-group $RG --name $APPNAME --query outboundIpAddresses --output tsv # Call this and add the IPs to the azure portal
-```
\ No newline at end of file
+```
+
+### Terraform
+
+```
+terraform apply -var "oauth_client_id=$OAUTH_CLIENT_ID" -var "oauth_client_secret=$OAUTH_CLIENT_SECRET" -var "secret_key=$SECRETKEY"
```

Comment:
When you identify the secret key issue, don't forget to update this line appropriately too